GVFS/GVFS.Common/Maintenance/PackfileMaintenanceStep.cs

```diff
@@ -127,6 +127,12 @@ protected override void PerformMaintenance()
                         activity.RelatedWarning($"Skipping {nameof(PackfileMaintenanceStep)} due to git pids {string.Join(",", processIds)}", Keywords.Telemetry);
                         return;
                     }
+
+                    if (this.Context.Repository.HasActiveLibGit2Repos)
+                    {
+                        activity.RelatedWarning($"Skipping {nameof(PackfileMaintenanceStep)} due to active libgit2 repos", Keywords.Telemetry);
```

Comment:
Can you remind me when we the `Telemetry` keyword is appropriate? I have some similar code, and now I am wondering if I need this keyword...

Reply:
We use it if we want to ensure it is sent to the telemetry stream. Other warnings are only written to logs. This way we can see if any users are always blocked on libgit2.